### DISCUSSION TITLE
WHAT HAVE YOU DONE

### DISCUSSION THREAD

**Discussion by Dvurechensky** [P0cL4bs/wifipumpkin3#244] [Q&A] [answered]
### Kali Linux 
- **Kernel: Linux 5.15.44-Re4son-v7l+**
- **Release:        2023.3**

```bash
 apt install wifipumpkin3
```

First start
```bash
wifipumpkin3
```
Result
![image](https://github.com/P0cL4bs/wifipumpkin3/assets/46356631/9cf7232e-4de3-45e8-bfa9-37e335ef7c59)

Second Start
```bash
wifipumpkin3
```
Result
![image](https://github.com/P0cL4bs/wifipumpkin3/assets/46356631/212306df-c47b-4fc7-94f1-d70195fdfc16)

WHAT HAVE YOU DONE?

**Comment by mh4x0f** [comment]
I follow the same process on kali 23. and installed without no  problem.

![image](https://github.com/P0cL4bs/wifipumpkin3/assets/6795613/b057a740-9206-4842-b0da-51171d7a6005)

try to remove and install again maybe can solve the problem.

**Reply by Dvurechensky** [reply]
Take a close look at the introductory system, Kali 2023 on ARM - Raspberi Pi, do I look like an idiot to you? The system was launched with the source code wifipumpkin3 and with the main package, always the same, typical errors were launched. I believe that whoever wrote the code for this product made it in such a way that it is impossible to separate the kernel from the PyQT5 module. At the moment, no matter who answers, I have written my own large set of programs for creating fake access points and NOT use wifipumpkin3 as a penetration testing tool.

**Reply by Dvurechensky** [reply]
Instead of taking the Kali Linux version with the Kernel I specified, you are seriously referring to the basic Kali Linux virtual machine with a different kernel. The level is clear.

**Reply by Dvurechensky** [reply, marked as answer]
The very first line in my discussion in an online search returns this link - https://www.kali.org/blog/secure-kali-raspberry-pi/ - this is how impudent you have to be to write something like that in response to me.
